core/src/value/partial.rs

```diff
@@ -173,42 +183,45 @@ enum DicomTimeImpl {
     Fraction(u8, u8, u8, u32, u8),
 }
 
-/// Represents a Dicom DateTime value with a partial precision,
+/// Represents a Dicom date-time (DT) value with a partial precision,
 /// where some date or time components may be missing.
 ///
-/// `DicomDateTime` is always internally represented by a [DicomDate]
-/// and optionally by a [DicomTime].
+/// `DicomDateTime` is always internally represented by a [DicomDate].
+/// The [DicomTime] and a timezone [FixedOffset] values are optional.
 ///
-/// It implements [AsRange] trait and also holds a [FixedOffset] value, from which corresponding
-/// [datetime][DateTime] values can be retrieved.
+/// It implements [AsRange] trait, which serves to retrieve a [PreciseDateTimeResult] from values
+/// with missing components.
```

Comment:
cargo doc cannot resolve `PreciseDateTimeResult` in this point, so we need to give it some directions.

```suggestion
/// It implements [AsRange] trait,
/// which serves to retrieve a [PreciseDateTimeResult](crate::core::value::PreciseDateTimeResult)
/// from values with missing components.
```

(with the rename, it would be this)

```suggestion
/// It implements [AsRange] trait,
/// which serves to retrieve a [PreciseDateTime](crate::core::value::PreciseDateTime)
/// from values with missing components.
```